backend/rates/gecko.go

```diff
@@ -16,21 +16,21 @@ const (
 	maxGeckoRange = 364 * 24 * time.Hour
 )
 
-// apiRateLimit specifies the minimal interval between equally spaced API calls
+// apiRateLimit specifies the maximum number of API calls per second
 // to one of the supported exchange rates providers.
-func apiRateLimit(baseURL string) time.Duration {
+func apiRateLimit(baseURL string) float64 {
```

Comment:
make the return type `rate.Limit` for clarity? then the callsite does not need to convert either.